docs/_docs/14-helpers.md

```diff
@@ -178,7 +178,8 @@ Embed a video from YouTube, Vimeo, or Google Drive that responsively sizes to fi
 | Parameter  | Required     | Description                                                |
 | ---------- | ------------ | ---------------------------------------------------------- |
 | `id`       | **Required** | ID of the video                                            |
-| `provider` | **Required** | Hosting provider of the video: `youtube`, vimeo`, or `google-drive` |
+| `provider` | **Required** | Hosting provider of the video: `youtube`, `vimeo`, `google-drive`, or `bilibili` |
+| `danmaku`  | optional     | bilibili only, [details below](###bilibili)                |
```

Comment:
The hash link requires *exactly* one hash sign, not three. See [this Stack Overflow answer](https://stackoverflow.com/a/2835151/5958455).

```diff
-| `danmaku`  | optional     | bilibili only, [details below](###bilibili)                |
+| `danmaku`  | optional     | bilibili only, [details below](#bilibili)                  |
```

Also a sentence should begin with an uppercase letter unless there's a convincing reason (not here I'd say).